### PR TITLE
Fixed unknown overlay 178 during TS map import

### PR DESCRIPTION
**Pull request by Mailaender**
See also https://ppmforums.com/viewtopic.php?p=530408 for reference.

### CODE REVIEW

File: OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs [hunk 1]
```diff
@@ -115,8 +115,8 @@ class ImportTSMapCommand : IUtilityCommand
 			{ 0xAF, "trock03" },
 			{ 0xB0, "trock04" },
 			{ 0xB1, "trock05" },
-			{ 0xBB, "veinholedummy" },
-			{ 0xBC, "crate" }
+
```
Comment (reaperrr) [inline OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs]: I'd say either remove the gap, or add a comment here, too.

Comment (Mailaender) [reply OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs]: Okay.

File: OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs [hunk 2]
```diff
@@ -115,8 +115,8 @@ class ImportTSMapCommand : IUtilityCommand
 			{ 0xAF, "trock03" },
 			{ 0xB0, "trock04" },
 			{ 0xB1, "trock05" },
-			{ 0xBB, "veinholedummy" },
-			{ 0xBC, "crate" }
+			//// Skipping the veinhole dummy on purpose.
```
Comment (pchote) [inline OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs]: Why four `/`'s?

**Review by pchote — COMMENTED** [review]
:+1: after fixing the comment.

File: OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs (hunk 2 above)
Comment (Mailaender) [reply OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs]: To silence StyleCop, which suggested this or add a newline for which the comment is the replacement on purpose.

Comment (pchote) [reply OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs]: A cleaner fix would be to replace it with `{ 0xB2, null }, // veinhole dummy` and then replace the hardcoded veinhole check in `ReadOverlay` with a null check to skip.

Comment (Mailaender) [reply OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs]: Moved the comment into the commit message.

**Review by reaperrr — APPROVED** [review]

**Pull request merged**